test/fixtures/test-runner/output/output.js

```diff
@@ -403,3 +404,41 @@ test('assertion errors display actual and expected properly', async () => {
     throw err;
   }
 });
+
+describe('should NOT print --test-only diagnostic warning - describe-only-false', {only: false}, () => {
```

Comment:
Can we put these changes in a new test. This fixture is run with a variety of reporters and makes the diff get really noisy. Some of these are already getting quite large and more difficult to debug as new changes are introduced.

Reply:
Hey @cjihrig, thanks for your super-fast feedback 🚀 
I moved the tests to a new file and added it to `test-runner-output.mjs`.

Hope that's fine!